clang/include/clang/AST/DeclTemplate.h

```diff
@@ -1435,7 +1436,10 @@ class NonTypeTemplateParmDecl final
   bool hasDefaultArgument() const { return DefaultArgument.isSet(); }
 
   /// Retrieve the default argument, if any.
-  Expr *getDefaultArgument() const { return DefaultArgument.get(); }
+  const TemplateArgumentLoc &getDefaultArgument() const {
```

Comment:
Is it better to rename `getDefaultArgument` to `getDefaultArgumentLoc`?

Reply:
All template parameters currently use that name, so this is kept for consistency.

I wouldn't object a later separate patch to rename all of them at once.